src/server/CMakeLists.txt

```diff
@@ -6,6 +6,17 @@ if(PDC_ENABLE_FASTBIT)
     find_library(FASTBIT_LIBRARY fastbit $ENV{HOME}/cori/fastbit-2.0.3/install)
 endif()
 
+if(PDC_ENABLE_ROCKSDB)
+    add_definitions(-DENABLE_ROCKSDB=1)
+    find_path(ROCKSDB_INCLUDE_DIR include/db.h)
+    find_library(ROCKSDB_LIBRARY rocksdb 8 REQUIRED)
+endif()
+
+if(PDC_ENABLE_SQLITE3)
+    add_definitions(-DENABLE_SQLITE3=1)
+    find_package(SQLite3 REQUIRED)
```

Comment:
Could you please include a check for a minimum version we support? Or at least have that in the documentation.

Reply:
done